src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -322,20 +322,52 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                             continue;
 
                         case "nullable":
+
+                            value = RemoveQuotesAndSlashes(value);
+                            if (value != null)
+                            {
+                                if (value.IsEmpty())
+                                {
+                                    AddDiagnostic(diagnostics, ErrorCode.ERR_SwitchNeedsString, MessageID.IDS_Text.Localize(), name);
+                                    continue;
+                                }
+                                switch (value.ToLower())
+                                {
+                                    case "disabled":
+                                        nullableContextOptions = NullableContextOptions.Disabled;
+                                        break;
+                                    case "enabled":
+                                        nullableContextOptions = NullableContextOptions.Enabled;
+                                        break;
+                                    case "safeonly":
+                                        nullableContextOptions = NullableContextOptions.SafeOnly;
+                                        break;
+                                    default:
+                                        AddDiagnostic(diagnostics, ErrorCode.ERR_BadNullableContextOption, value);
+                                        break;
+                                }
+                            }
+                            else
+                            {
+                                nullableContextOptions = NullableContextOptions.Enabled;
+                            }
+                            continue;
+
+
                         case "nullable+":
```

Comment:
>case "nullable+": [](start = 24, length = 17)

Should we remove `/nullable+` and `/nullable-` now that we have `/nullable:enabled` and `/nullable:disabled`?